packages/ocaml/ocaml.4.08.0/opam

```diff
@@ -12,10 +12,16 @@ depends: [
   "ocaml-system" {>= "4.08.0" & < "4.08.1~"}
 ]
 setenv: [
+  [OCAMLTOP_INCLUDE_PATH += "%{toplevel}%"]
   [CAML_LD_LIBRARY_PATH = "%{_:stubsdir}%"]
   [CAML_LD_LIBRARY_PATH += "%{lib}%/stublibs"]
+  # Legacy opam variable
   [OCAML_TOPLEVEL_PATH = "%{toplevel}%"]
 ]
+x-env-path-rewrite: [
+  [CAML_LD_LIBRARY_PATH (";" {os = "win32"} ":" {os != "win32"}) "target"]
+  [OCAMLTOP_INCLUDE_PATH (";" {os = "win32"} ":" {os != "win32"}) "target"]
```

Comment:
Any reason why you don't do this for `OCAML_TOPLEVEL_PATH` ?

Reply:
It's not a PATH-like variable - it's only ever set using `=`

Reply:
Ah, I see :–)